src/filtering/predict.jl

```diff
@@ -43,7 +43,10 @@ function predict!(
 end
 
 function predict_mean!(x_out::Gaussian, x_curr::Gaussian, Ah::AbstractMatrix)
-    mul!(x_out.μ, Ah, x_curr.μ)
+    D = length(x_out.μ)
```

Comment:
This and the below should probably be turned into some function, as this is really only there as a fast, allocation-free multiplication that potentially uses the Kronecker-trick if applicable (if `Ah`) is small.